### DISCUSSION TITLE
File Uploader to allow SVG images

### DISCUSSION THREAD

**Discussion by RoKrypto** [wintercms/winter#625] [Q&A]
Greetings.

I'm using the Builder Plugin to develop my project. While watching a videotutorial on how to use it, I realized some features of the File Uploader aren't available in comparison with the one shown in the video, like the type of file I'd like it to accept (such as "File" or "Image") and the formats of the selected type. Precisely, the latter prevents me from being able to upload SVG images. Is there a workaround for this?

**Comment by arvislacis** [comment]
I think it's related to https://github.com/wintercms/winter/issues/9 As you could potentially include all kind of stuff in SVG images then you need to sanitize the uploaded SVG image file (functionality which doesn't exist yet) but I think there may be workarounds if you really want to allow that and trust that uploaded images would not be malicious.

**Comment by arvislacis** [comment]
Actually @bennothommo has mentioned a workaround in one of comments of that issue - https://github.com/wintercms/winter/issues/9#issuecomment-1086989354 (https://wintertricks.com/tricks/enable-svg-support-in-media-manager)

**Reply by RoKrypto** [reply]
Hi @arvislacis . So, should I go with using the media manager instead of the file uploader? I'll take a look at the link you posted. Thanks!

**Reply by arvislacis** [reply]
@RoKrypto I think similar approach can be applied also to file uploader. But, yes, if you can then go with mentioned media manager solution.

**Reply by bennothommo** [reply]
I believe the changes suggested in the trick also apply to the file uploader as well.
